isis/tests/CMakeLists.txt

```diff
@@ -3,13 +3,16 @@ cmake_minimum_required(VERSION 3.10)
 add_dependencies(isis3 isis3)
 
 file(GLOB test_source "${CMAKE_SOURCE_DIR}/tests/*.cpp")
-file(GLOB isis_libs "${CMAKE_BINARY_DIR}/lib/lib*")
+
+# Non-ideal temporary solution to link against other shared libraries for mission tests, etc
+file(GLOB isis_libs "${CMAKE_BINARY_DIR}/lib/*.so")
+file(GLOB isis_mac_libs "${CMAKE_BINARY_DIR}/lib/*.dylib")
```

Comment:
oof

Reply:
I'm agreeing with the oof. Putting in a ticket to actually fix this later.

Reply:
This looks like it will forcibly link against all libraries including the plugins that are not supposed to be loaded at initial run time. May not actually test the plugins correctly. As long as the new ticket has this documented, I'm ok with it for now.